modules/codepipeline/templates/buildspec_build.yml

```diff
@@ -13,7 +13,7 @@ phases:
       - echo `ls -lrt`
       - echo Build started on `date`
       - echo Building the Docker image...          
-      - docker build -t $IMAGE_URI .
+      - docker build -t $IMAGE_URI lambda/
```

Comment:
This was a bug in the provided lab. Since the Dockerfile exists underneath the `lambda` directory, we need our buildspec to correctly locate it.